ci-operator/jobs/terraform-redhat/terraform-rhcs-rosa/terraform-redhat-terraform-rhcs-rosa-main-presubmits.yaml

```diff
@@ -0,0 +1,149 @@
+presubmits:
```

Comment:
why not rehearsing the new jobs? (seeing you've already committed for "pj-rehearse ack")